lms/events/event.py

```diff
@@ -66,12 +66,7 @@ def _get_role_ids(self):
         return [role.id for role in self.request.lti_user.lti_roles]
 
     def _get_application_instance_id(self):
-        if application_instance := self.request.find_service(
-            name="application_instance"
-        ).get_current():
-            return application_instance.id
-
-        return None
+        return self.request.lti_user.application_instance_id
```

Comment:
This could've been done before in isolation, we already had the ai.id in the lti_user.